### DISCUSSION TITLE
When you rename an user, the modification isn't applied globally

### DISCUSSION THREAD

**Discussion by tigerblue77** [requarks/wiki#3308] [Error / Bug Report]
**Describe the bug**
If I'm logged as "Administrator" (for example) and rename my user account "admin" :
- when I visit a page that I edited the last, I still see "Administrator"
- when I click on my profile (top right button, in header) I still see "Administrator" and not my new name "admin"

**To Reproduce**
1. Go to 'Administration zone'
2. Click on 'Users'
3. Click on 'Administrator'
4. Edit 'Full name' to what you want
5. Check a page you edited the last
6. Click on your profile (top right button, in header)

**Expected behavior**
I expected my name to change everywhere

**Host Info (please complete the following information):**
- OS: Windows 10 Professionnel Insider Preview (20H2 Build 21292.1000)
- Wiki.js version: 2.5.170
- Database engine: Postgres 13

**Comment by tigerblue77** [comment]
Hello, I've upgraded my Wiki.js to 2.5.191 and this problem is still present.

**Reply by NGPixel** [reply]
This has already been reported as #982
